examples/LoadingData.md

```diff
@@ -0,0 +1,114 @@
+
+# Loading Data
+
+The most basic task you can do in the analysis schema is load your data. This is done by calling the keyword `Data` and filling out the `FileName` and `DatasetName` attributes. The `FileName`  is the file or file path for your data and the `DatasetName` is the name (in the form of a string) for your data, similar to a variable name. 
```

Comment:
While you could submit a config file that just loads data, it would run and not actually return any output to the user... so I think this should be rephrased as more of a  fundamental step in building workflows rather than a standalone task.

Reply:
Done!